lib/Core/Executor.cpp

```diff
@@ -194,73 +157,84 @@ namespace {
                            clEnumValN(ExternalCallPolicy::Concrete, "concrete", "Only external function calls with concrete arguments are allowed (default)"),
                            clEnumValN(ExternalCallPolicy::All, "all", "All external function calls are allowed.  This concretizes any symbolic arguments in calls to external functions.")
                            KLEE_LLVM_CL_VAL_END),
-                cl::init(ExternalCallPolicy::Concrete));
+                cl::init(ExternalCallPolicy::Concrete),
+                cl::cat(ExtCallsCat));
+
+  cl::opt<bool>
+  SuppressExternalWarnings("suppress-external-warnings",
+			   cl::init(false),
+			   cl::desc("Supress warnings about calling external functions."),
+                           cl::cat(ExtCallsCat));
+
+  cl::opt<bool>
+  AllExternalWarnings("all-external-warnings",
+		      cl::init(false),
+		      cl::desc("Issue a warning everytime an external call is made," 
+			       "as opposed to once per function (default=off)"),
+                      cl::cat(ExtCallsCat));
+
+
+  /*** Seeding options ***/
+
+  cl::OptionCategory SeedingCat("Seeding options",
+                                "These options are related to the use of seeds to start exploration.");
 
   cl::opt<bool>
   AlwaysOutputSeeds("always-output-seeds",
-		    cl::init(true));
+		    cl::init(true),
+                    cl::desc("(default=true)"),
+                    cl::cat(SeedingCat));
 
   cl::opt<bool>
   OnlyReplaySeeds("only-replay-seeds",
 		  cl::init(false),
-                  cl::desc("Discard states that do not have a seed (default=off)."));
- 
+                  cl::desc("Discard states that do not have a seed (default=off)."),
+                  cl::cat(SeedingCat));
+
   cl::opt<bool>
   OnlySeed("only-seed",
 	   cl::init(false),
-           cl::desc("Stop execution after seeding is done without doing regular search (default=off)."));
+           cl::desc("Stop execution after seeding is done without doing regular search (default=off)."),
+           cl::cat(SeedingCat));
  
   cl::opt<bool>
   AllowSeedExtension("allow-seed-extension",
 		     cl::init(false),
-                     cl::desc("Allow extra (unbound) values to become symbolic during seeding (default=false)."));
+                     cl::desc("Allow extra (unbound) values to become symbolic during seeding (default=false)."),
+                     cl::cat(SeedingCat));
  
   cl::opt<bool>
   ZeroSeedExtension("zero-seed-extension",
 		    cl::init(false),
-		    cl::desc("(default=off)"));
+		    cl::desc("(default=off)"),
```

Comment:
```suggestion
		    cl::desc("Use zero-filled objects if matching seed not found (default=off)"),
```

Reply:
This one can be still added.

Reply:
Hi @MartinNowack , your comment is on the old code